FirebaseRemoteConfigSwift/Sources/PropertyWrapper/PropertyWrapper.swift

```diff
@@ -1,59 +1,79 @@
-//
-//  PropertyWrapper.swift
-//  
-//
-//  Created by Fumito Ito on 2022/05/25.
-//
+/*
+ * Copyright 2021 Google LLC
```

Comment:
This and following copyrights should be 2022.

Reply:
Thanks ! I fixed it at https://github.com/firebase/firebase-ios-sdk/pull/9918/commits/a491ac1d8be2b2abe26938e1aa2329b0b67a04ff.